image/templates/helm/stackrox-central/values.yaml.htpl

```diff
@@ -538,7 +538,12 @@ central:
 #    # sufficient and manual override is required.
 #    configOverride: null
 #
+[< if .FeatureFlags.ROX_POLICY_AS_CODE >]
```

Comment:
Interesting there is a feature flag check here before setting the values as true which is the correct thing to do, but there is the config yaml where we don't check feature flag. We should for both correctness and consistency right?

Reply:
Are you referring to `config-share.yaml`?

Reply:
`[<- .... >]`

Reply:
ah, it turns out that the config-shape.yaml file can't use the brackets because it's not a meta template (`.htpl`) file.